contributed_definitions/NXexperimentdata.nxdl.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+# 
+# Copyright (C) 2012-2017 NeXus International Advisory Committee (NIAC)
+# 
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+
+<definition name="NXexperimentdata" extends="NXobject" type="group"
+	    version="1.0"  
+	    category="contributed"
+	    xmlns="http://definition.nexusformat.org/nxdl/3.1"
+	    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+	    >
+	<doc>
+		Describe all the plottable data from a single detector in a single entity. To be used
+		inside :ref:`NXexperiment`
+		
+		Allows data to be related back to the detector that collected it.
+		
+		(**required**) :ref:`NXdetector` detector used to acquire data.
+		
+		(**required**) :ref:`NXdata` at least one NXdata, related to the NXdetector
+  </doc>
+  <attribute name="DATANAME_origin">
+    <doc>
+      Best description of the origin of this data, use . for the :ref:`NXdetector`. ``DATANAME`` refers to the 
+      name of an :ref:`NXdata` that must be in the same :ref:`NXexperimentdata`.
+    </doc>
+  </attribute>
+  <attribute name="primary">
```

Comment:
change name of this attribute to `default` (per #380)  

documentation should mention that an absolute HDF5 path address (starts with `/`, such as `/entry/instrument/detector1/ccd3`) must be used.